src/Client.php

```diff
@@ -4,15 +4,83 @@
 
 namespace Answear\FocusContactCenterBundle;
 
+use Answear\FocusContactCenterBundle\Exception\ApiError;
+use Answear\FocusContactCenterBundle\Exception\MalformedResponse;
+use Answear\FocusContactCenterBundle\Exception\ServiceUnavailable;
+use GuzzleHttp\Exception\GuzzleException;
+use GuzzleHttp\RequestOptions;
+use Webmozart\Assert\Assert;
+
 class Client
 {
+    /**
+     * @var Configuration
+     */
+    private $configuration;
+
+    /**
+     * @var HashGenerator
+     */
+    private $hashGenerator;
+
+    /**
+     * @var \GuzzleHttp\Client
+     */
+    private $guzzle;
+
+    public function __construct(Configuration $configuration, HashGenerator $hashGenerator, \GuzzleHttp\Client $guzzle)
+    {
+        $this->configuration = $configuration;
+        $this->hashGenerator = $hashGenerator;
+        $this->guzzle = $guzzle;
+    }
+
     public function fccAddRecords(Request\FCC\AddRecords $request): Response\FCC\AddRecords
     {
-        // TODO
+        return Response\FCC\AddRecords::fromArray($this->request('fcc-add-records', $request));
     }
 
     public function fccUpdateRecords(Request\FCC\UpdateRecords $request): Response\FCC\UpdateRecords
     {
-        // TODO
+        return Response\FCC\UpdateRecords::fromArray($this->request('fcc-update-records', $request));
+    }
+
+    private function request(string $endpoint, Request\Request $request): array
+    {
+        $change = $this->configuration->getChangeIdGenerator()->generate();
+        $auth = [
+            'login' => $this->configuration->getLogin(),
+            'change' => $change,
+            'hash' => $this->hashGenerator->hash($change),
+            'method' => $this->configuration->getHashMethod(),
+            'campaigns_id' => $this->configuration->getCampaignsId(),
+        ];
+        try {
+            $response = $this->guzzle->request(
+                'POST',
+                sprintf('%s/%s', $this->configuration->getUrl(), $endpoint),
+                [
+                    RequestOptions::JSON => \array_merge($auth, $request->toArray()),
+                ]
+            );
+            $responseText = $response->getBody()->getContents();
+        } catch (GuzzleException $e) {
+            throw new ServiceUnavailable($e->getMessage(), $e->getCode(), $e);
+        }
+
+        $decoded = \json_decode($responseText, true);
```

Comment:
Consider to add `JSON_THROW_ON_ERROR` option

Reply:
`JSON_THROW_ON_ERROR` was added in PHP 7.3 while this package has minimum requirement of 7.1